pkg/driver/driver_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package driver_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestCosiDev(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "CosiDev Suite")
```

Comment:
TODO: update this to Driver Suite
```suggestion
	RunSpecs(t, "Driver Suite")
```